README.md

```diff
@@ -260,6 +261,13 @@ for Modeling Source Code?](http://web.cs.ucdavis.edu/~devanbu/isDLgood.pdf) - Vi
 * [Embedding the GitHub contribution graph](https://egorbu.github.io/techtalks-2017-moscow)
 * [Measuring code sentiment in a Git repository](http://vmarkovtsev.github.io/gophercon-2018-moscow/)
 
+### Video
+
+* ["Neural Abstract Machines & Program Induction"]( https://www.youtube.com/playlist?list=PLzTDea_cM27LVPSTdK9RypSyqBHZWPywt)  Workshop at NIPS 2016
+* ["Learning to Code: Machine Learning for Program Induction"](https://www.youtube.com/watch?v=vzDuVhFMB9Q) CamAIML 2017 presentation by MSR.
```

Comment:
Maybe it's worth to add `CamAIML` conference + talks because they have one more related talk
```
Learning to Code: Machine Learning for Program Induction | Alex Gaunt 
Learning Program Representations: Symbols to Vectors to Semantics | Miltos Allamanis```.
What do you think? In this case, we can preserve structure as it is for now.